mobile/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
                   android:required="false"/>
     <uses-feature android:name="android.software.leanback"
                   android:required="false"/>
-    <uses-feature android:name="android.hardware.camera"
+    <uses-feature android:name="android.hardware.camera"        
```

Comment:
why space here?

Reply:
https://github.com/shadowsocks/shadowsocks-android/pull/1446/commits/6678fd9541e280963d6680d567c12c4d69e846ef
🤷‍♂️

Reply:
@madeye can handle it if he decides to merge